From 52e3b4296322ea479cbee1b02de435cc00675d50 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Fri, 24 Nov 2023 08:56:41 +0100
Subject: [PATCH 0094/1695] dts: stm32mp15: enable secure IWDG1 on ST boards

Enable IWDG1 watchdog device in ST boards stm32mp157a-dk1,
stm32mp157c-dk2, stm32mp157c-ed1 and stm32mp157c-ev1.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 16728f2f547780fb5cda79f47d372440bfab1166 dts: stm32mp15: enable secure IWDG1 on ST boards]
Change-Id: I06d4c068faa5e6a47ab87aa40b5ff6df7f4fd342
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
[etienne: cherry-picked from 23ca21385d22d70f97dbdf61f2cbef415515e9ae]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/dts/stm32mp157c-ed1.dts  | 1 +
 core/arch/arm/dts/stm32mp15xx-dkx.dtsi | 1 +
 2 files changed, 2 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp157c-ed1.dts b/core/arch/arm/dts/stm32mp157c-ed1.dts
index eb679f5db..8663287ab 100644
--- a/core/arch/arm/dts/stm32mp157c-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157c-ed1.dts
@@ -314,6 +314,7 @@
 
 &iwdg1 {
 	timeout-sec = <32>;
+	status = "okay";
 };
 
 &iwdg2 {
diff --git a/core/arch/arm/dts/stm32mp15xx-dkx.dtsi b/core/arch/arm/dts/stm32mp15xx-dkx.dtsi
index a0d4eb20b..87f853b3f 100644
--- a/core/arch/arm/dts/stm32mp15xx-dkx.dtsi
+++ b/core/arch/arm/dts/stm32mp15xx-dkx.dtsi
@@ -452,6 +452,7 @@
 
 &iwdg1 {
 	timeout-sec = <32>;
+	status = "okay";
 };
 
 &iwdg2 {
-- 
2.47.2

